public/index.html

```diff
@@ -2,7 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+
+    <!-- Old font here -->
     <link rel="stylesheet" href="https://use.typekit.net/xld2lxj.css">
```

Comment:
Are we still using the old font anywhere? Might be best to comment this line out so that we can save a bit on bandwidth. Not that it'd probably save that much time, it already takes a long time to load.